src/pages/edge/adobe-journey-optimizer/proposition-edge-events.md

```diff
@@ -0,0 +1,354 @@
+                "eventType" : "personalization.request"
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionTrigger
+
+An event similar to the following is emitted any time a user has met the client-side criteria that qualifies them for a proposition:
+
+```json
+{
+    "events": [
+        {
+            "xdm": {
+                "_id": "F8B3C334-A67D-4174-A0F2-4E00523F5D02",
+                "eventType": "decisioning.propositionTrigger",
+                "timestamp": "2024-09-25T19:48:16.092Z",
+                "_experience": {
+                    "decisioning": {
+                        "propositionEventType": {
+                            "trigger": 1
+                        },
+                        "propositions": [
+                            {
+                                "id": "4b6fe796-3e4c-476d-be62-8610d851daef",
+                                "scope": "mobileapp://com.my.app.bundleIdentifier",
+                                "scopeDetails": {
+                                    "characteristics": {
+                                        "eventToken": "<BASE64_ENCODED_STRING>"
+                                    },
+                                    "activity": {
+                                        "id": "6aebeb06-faeb-4864-a27c-8be3efd6ff9b#876cb49a-7094-4f9e-9f9d-57ee26858462",
+                                        "matchedSurfaces": [
+                                            "mobileapp://com.my.app.bundleIdentifier/"
+                                        ]
+                                    },
+                                    "correlationID": "7d0408d2-e9ed-4d66-b0a5-2659fc37e2a9-0",
+                                    "decisionProvider": "AJO"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    ]
+}
+```
+
+### decisioning.propositionDisplay
+
+An event similar to the following is emitted any time a proposition is displayed to the application's UI:
+
+```json
+{
+    "events": [
+        {
+            "xdm": {
+                "_id": "F8B3C334-A67D-4174-A0F2-4E00523F5D02",
+                "eventType": "decisioning.propositionDisplay",
+                "timestamp": "2024-09-25T19:48:16.092Z",
+                "_experience": {
+                    "decisioning": {
+                        "propositionEventType": {
+                            "display": 1
+                        },
+                        "propositions": [
+                            {
+                                "id": "4b6fe796-3e4c-476d-be62-8610d851daef",
+                                "scope": "mobileapp://com.my.app.bundleIdentifier",
+                                "scopeDetails": {
+                                    "characteristics": {
+                                        "eventToken": "<BASE64_ENCODED_STRING>"
```

Comment:
There's another `subPropositions` field that is sent in the personalization response for CBE which needs to be echoed back as is in the display and interact tracking events. The wiki  I shared above has it in the payload